velox/docs/monthly-updates/august-2022.rst

```diff
@@ -0,0 +1,80 @@
+******************
+August 2022 Update
+******************
+
+Documentation
+=============
+
+* Add documentation for :func:`combinations` Presto function.
+* Add documentation for :doc:`/functions/window`.
+* Add documentation for :func:`array_sort`.
+* Add `website`_ for Velox.
+
+.. _website: https://velox-lib.io
+
+Core Library
+============
+
+* Add support for Right Semi Join.
+* Add support for a simple in-memory Window operator.
+* Add support for `month` string in Joda library.
+* Add support for DECIMAL arithmetic addition, subtraction functions.
+* Add support for spilling `order by` to disk.
+* Add support for zero-copy vector view.
+* Add support for casting DECIMAL type to DECIMAL type.
+* Add support for DECIMAL type sum aggregation.
+* Improve spilling to disk by avoiding redundant computations, better test coverage.
+* Resolve to vector functions over simple functions when their signatures match.
+* Pre-load lazy vectors that are referenced by multiple sub-expressions.
+
+Core Extensions
```

Comment:
@mbasmanova  I add a new section in this report to cover Substrait and Arrow extensions. What do you think?

Reply:
Perhaps, split it into two: Substrait and Arrow.